packages/default-theme/src/pages/checkout.vue

```diff
@@ -164,7 +164,13 @@ export default {
       if (!isFormCorrect) {
         return
       }
-      await register(registrationFormData.value)
+
+      // register user and point /checkout path that may be used in double opt-in (optional usage)
+      await register(
+        Object.assign({}, registrationFormData.value, {
+          redirectTo: encodeURIComponent(PAGE_CHECKOUT),
```

Comment:
we should use this for url, to save domain settings
```suggestion
          redirectTo: routing.getUrl(PAGE_CHECKOUT),
```

`routing` object can be fetched from `getApplicationContext`

Reply:
thanks, hawk-eye! without routing.getUrl we are always redirected to the default language without any prefix 🙏🏻 now it's fixed. I left encodeURIComponent, path is automatically decoded on vue side 👍🏻